docs/content/preview/reference/configuration/yb-tserver.md

```diff
@@ -1003,7 +1003,9 @@ In addition, as this setting does not propagate to PostgreSQL, it is recommended
 --ysql_pg_conf_csv="ssl_min_protocol_version=TLSv1.2"
 ```
 
-## Packed row flags (Beta)
+## Packed row flags
+
+Packed row format support is currently in [Early Access](/preview/releases/versioning/#feature-availability).
 
 To learn about the packed row feature, see [Packed row format](../../../architecture/docdb/persistence/#packed-row-format-beta) in the architecture section.
```

Comment:
This link needs to be updated in a couple of places. Can you search for #packed-row-format-beta and fix them all?